IOPool/Output/src/PoolOutputModule.cc

```diff
@@ -483,10 +484,14 @@ namespace edm {
         ->setComment(
             "True:  Allow fast copying, if possible.\n"
             "False: Disable fast copying.");
+    desc.addUntracked("mergeJob", false)
+        ->setComment(
+            "If set to true and fast copying is disabled, copy input file compression and basket sizes to the output "
+            "file.");
```

Comment:
I'm thinking if more descriptive name would be clearer (e.g. along monstrous `useCompressionBasketSizeDefaultsFromInput`)?

Or do you see the usage more of "now we know the job is a merge job, and can do various optimizations based on that"? (the code hints a bit towards that)

Reply:
Indeed, I think there's more that could be done to optimize merge jobs where fast-cloning isn't possible, so I'd rather make this a hint to the output module, not a specific directive.